client/src/components/Error404.style.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--highlight);
+  height: 100vh;
+  box-sizing: border-box;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+`;
+export const Div = styled.div`
+  text-align: left;
```

Comment:
I think you can move this into the styles for the main tag - it doesn't seem to do much and so is an unnecessary div

Reply:
mmm that was the last solution to get the style I want it , if u have an alternative that would be great.